core/templates/pages/error-pages/error-page.component.ts

```diff
@@ -32,9 +32,12 @@ angular.module('oppia').component('errorPage', {
       ctrl.getStatusCode = function() {
         return Number(ctrl.statusCode);
       };
+
+      ctrl.getStaticImageUrl = function(imagePath) {
```

Comment:
Why are these changes in this PR?

Reply:
My last PR server images as webp got merged but i guess due to some merge conflicts this part got changed in that PR. PR link: #9206

Reply:
@nithusha21 PTAL!

Reply:
If that is the case, can you remove it from here? Or merge from develop?

Reply:
Move it from here and make a new PR? @nithusha21